src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final int DEFAULT_AGE = 27;
+    private static final String DEFAULT_LOGIN = "curvabobr";
+    private static final String DEFAULT_PASSWORD = "Defaultpassword17";
+    private static final int MINIMAL_AGE = 18;
+    private static final int MINIMAL_PASSWORD_LENGTH = 6;
+    private static final int MINIMAL_LOGIN_LENGTH = 4;
+    private static final int FIRST_LOGIN_CHARACTER = 0;
+
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private final StorageDao storageDao = new StorageDaoImpl();
+    private User user;
```

Comment:
I have a question for mentor mostly - isnt this better to create separate Test class to test StorageDaoImpl separately?

Reply:
we do not need to test storage in this task, overall dao classes are tested only if there is some custom logic, but in this case it is just a simulation, so no need to tests it